safari/Refined GitHub/Refined GitHub/ViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ViewController.swift
+//  Refined GitHub
+//
+//  Created by Rico on 21/10/20.
+//
+
+import Cocoa
+import SafariServices.SFSafariApplication
+import SafariServices.SFSafariExtensionManager
+
+let appName = "Refined GitHub"
+let extensionBundleIdentifier = "com.sindresorhus.Refined-GitHub-Extension"
```

Comment:
```suggestion
let extensionBundleIdentifier = "com.sindresorhus.Refined-GitHub"
```

Since that's the bundle ID I registered.